packages/testing/src/server.ts

```diff
@@ -26,12 +26,12 @@ const setTimeoutPromise = promisify(setTimeout)
 // response: json payload response (defaults to {})
 // status: http status code (defaults to 200)
 // wait: number used to induce a certain time delay in milliseconds in the response (defaults to undefined)
-export const startServer = async (handler: ServerHandler) => {
+export const startServer = async (handler: ServerHandler, port = 0) => {
```

Comment:
Sometimes it's useful to know the port before starting the server, so this makes it possible for the consumer to specify the port that the server should use.